env/edit.go

```diff
@@ -6,11 +6,13 @@ import (
 	"fmt"
 	"regexp"
 	"strings"
+	"os"
 
 	"gopkg.in/errgo.v1"
 
 	"github.com/Scalingo/cli/config"
 	"github.com/Scalingo/go-scalingo/v6"
+	"github.com/joho/godotenv"
```

Comment:
I am kind of reluctant to introduce an external dependency for this use case. What do you think about this @Soulou?

Reply:
@EtienneM Actually, it doesn't bring a lot of value to re-implement it and we can understand the need related to this PR.

The question is more: is the dependency safe enough?

* Is it maintained? (Number of releases, developers, etc.)
* Is CI configured?
* Quick audit of the code base

To me these signals are green-ish, and would go for it.

Reply:
I'm not sure I agree with these arguments but you are the boss :p

This dependency seems well maintained with good programming practice, let's go :rocket: